src/components/BookStorage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Book from './Book';
+
```

Comment:
- **[OPTIONAL]** For structure organizing I recommend separating pages from components by creating a `src/pages` directory that will contain both <BookStorage  />  and <Categories /> pages👍🏼

Reply:
Thanks for your feedback. I will consider this